### PR TITLE
[KV]: add kv vs d1

### PR DESCRIPTION
**Pull request by thomasgauvin**
@vy-ton what do you think?

### CODE REVIEW

**Review by rts-rob — APPROVED** [review]

File: src/content/partials/kv/kv-vs-d1.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+{}
+---
+
+## Workers KV vs D1
+
+Cloudflare Workers KV provides an eventually consistent global key-value store that caches data throughput Cloudflare's network to provide
```
Comment (vy-ton) [inline src/content/partials/kv/kv-vs-d1.mdx]:
```suggestion
Cloudflare Workers KV provides an eventually consistent global key-value store that caches data throughout Cloudflare's network to provide
```

File: src/content/partials/kv/kv-vs-d1.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+low read latency for hot reads to keys. This is ideal for storing data that is repeatedly read by your Workers, such as configuration data, user preferences, cached values, etc. Workers KV can sustain high read throughput (unlimited requests per second per key) with \<5ms latency globally for hot reads. Workers KV is eventually consistent, so writes may take up to 60 seconds to propagate through Cloudflare's network by default.
+
+Cloudflare D1 provides a SQL database that supports relational data modeling and quering. D1 supports snapshot isolation consistency and is ideal for
```
Comment (vy-ton) [inline src/content/partials/kv/kv-vs-d1.mdx]:
```suggestion
Cloudflare D1 provides a SQL database that supports relational data modeling and querying. D1 supports snapshot isolation consistency and is ideal for
```

File: src/content/partials/kv/kv-vs-d1.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+workloads that store user data or general web application data. D1 currently stores data regionally (with read replicas coming soon).
```
Comment (vy-ton) [inline src/content/partials/kv/kv-vs-d1.mdx]:
```suggestion
workloads that store user data or general web application data. 
```

Let's avoid mentioning future feature release

File: src/content/partials/kv/kv-vs-d1.mdx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+- Use Workers KV if you need to store and access configuration data with that will be read by Workers frequently, that is written infrequently (\<1 RPS per key) and can tolerate eventual consistency.
```
Comment (vy-ton) [inline src/content/partials/kv/kv-vs-d1.mdx]:
```suggestion
- Use Workers KV if you need to store and access configuration data that will be read by Workers frequently, is written infrequently (\<1 RPS per key) and can tolerate eventual consistency.
```

**Review by vy-ton — APPROVED** [review]

**Review by Oxyjun — APPROVED** [review]

**Pull request merged**